src/main/java/Lession3/Seat.java

```diff
@@ -0,0 +1,31 @@
+package Lession3;
+
+public class Seat {
+    private final int seatNumber;
+    private final int rowNumber;
+
+    private boolean occupied;
+
+    public Seat(int seatNumber, int rowNumber) {
+        this.seatNumber = seatNumber;
+        this.rowNumber = rowNumber;
+        this.occupied=false;
+
+    }
+
+    public int getSeatNumber(){
+        return seatNumber;
+
+    }
+
+    public int getRowNumber(){
+        return rowNumber;
+    }
+
+    public boolean isOccupied() {
+        return occupied;
+    }
+
```

Comment:
It would be great to create some method in which we will be able to change occupied parameter value. Those kind of methods are called setters, here You have some explanation about it https://www.w3schools.com/java/java_encapsulation.asp Please add a setter for occupied variable.